GDJS/Runtime/howler-sound-manager/howler-sound-manager.ts

```diff
@@ -655,7 +766,7 @@ namespace gdjs {
     }
 
     getSoundOnChannel(channel: integer): HowlerSound {
```

Comment:
What do you think of changing the return type to `HowlerSound | null`, rather than using the null object pattern? This might be clearer for the user - better fail early than later?

Reply:
Ok